### PR TITLE
added 'Ignore', 'InvertMatch' blocks to snmp.conf.erb and the snmp da…

### PR DESCRIPTION
**Pull request by lilhuang**
Please check the following items before submitting a PR -- thank you!

Note that this project is released with a Contributor Code of Conduct.
By participating in this project you agree to abide by its terms.
[Contributor Code of Conduct](https://voxpupuli.org/coc/).

- [x] There is no existing PR that addresses this problem
- [x] Mentioned any existing issues in your commit so they get linked and
  closed once this PR gets merged, i.e: `Closes #1554` in the body of a commit
- [x] Followed the instructions in the [Contributing](CONTRIBUTING.md) document
- [x] Ran the unit/spec tests and ensured they still pass
- [x] Added tests to cover the new behaviour
- [x] Updated the documentation to match the changes
- [x] When possible, add an entry to the CHANGELOG file
- [x] Squashed your PR down to a single commit. You may forego this if the PR
  tries to address multiple issues. Though we prefer one PR per feature/fix,
  sometimes that's not feasible. In that case ensure that a single feature/fix
  and associated tests and documentation is bundled up in one commit

Optional, but extra points:

- [ ] Added tests to ensure the old behaviour cannot accidentally be
  reintroduced

- - -

…ta.conf.erb file, added ignore and invertmatch vars to collectd::plugin::snmp::data, as well as appropriate tests and documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1304,10 +1304,12 @@ class {'collectd::plugin::sensors':
...
     amavis_incoming_messages => {
-      'Type'     => 'counter',
-      'Table'    => false,
-      'Instance' => 'amavis.inMsgs',
-      'Values'   => ['AMAVIS-MIB::inMsgs.0']
+      'Type'        => 'counter',
+      'Table'       => false,
+      'Instance'    => 'amavis.inMsgs',
+      'Values'      => ['AMAVIS-MIB::inMsgs.0']
+      'Ignore'      => '00:00'
```
Comment (ffrank) [inline README.md]: I honestly don't know how `Ignore` works in this context. Is `00:00` a good example?

**Comment by bastelfreak** [comment]
Hi @lilhuang, can you please take a look at the comment from @ffrank?

File: README.md (hunk 1 above)
Comment (bmeekhof) [inline README.md]: Hi, I've taken over responsibility from @lilhuang for this.  I believe 00:00 is a good example.  In our context, it is what SNMP returns for an empty switch interface alias so we want to ignore it - it seems reasonable that others may encounter a similar situation.  I've updated the commit to README.md in the pull request to also include an example of wildcard matching a string and putting multiple Ignore values in an array.  

`+      'Ignore'      => [ '00:00', '*IgnoreString' ],`

**Comment by ffrank** [comment]
Fine by me. 👍 
Rubocop is complaining. Guess that can be fixed in post.

**Pull request merged**